src/Symfony/Component/Lock/StoreInterface.php

```diff
@@ -39,7 +39,7 @@ public function save(Key $key);
      * @throws LockConflictedException
      * @throws NotSupportedException
      */
-    public function waitAndSave(Key $key);
+    public function waitAndSave(Key $key): void;
```

Comment:
Adding return types in an interface is a BC break, and so cannot be done until 5.0

Reply:
@stof if I didn't make changes to interfaces but only on concrete implementations, will that help get these changes out faster? And I can put up a separate PR for `5.0` version.